code/game/objects/items/tools/powertools.dm

```diff
@@ -67,8 +67,8 @@
 
 	tool_act_off = TOOL_SCREWDRIVER
 	tool_act_on = TOOL_WRENCH
-	action_off = "screw driver"
-	action_on = "bolt driver"
+	action_off = "bolt driver"
+	action_on = "screw driver"
```

Comment:
I think the changes to `action_off` and `action_on` are unnecessary, and instead the `balloon_alert` ternary needs to fixed

Reply:
Ditto the above, confirmed to be fine per my own testing.